picard/ui/mainwindow.py

```diff
@@ -902,6 +903,7 @@ def update_actions(self):
         self.cut_action.setEnabled(bool(self.selected_objects))
         files = self.get_selected_or_unmatched_files()
         self.tags_from_filenames_action.setEnabled(bool(files))
+        self.track_search_action.setEnabled(have_objects)
```

Comment:
`self.track_search_action.setEnabled(bool(self.selected_objects))` and drop `have_objects` ?